ios/REANodesManager.m

```diff
@@ -128,6 +117,12 @@ - (void)postRunUpdatesAfterAnimation
 - (void)startUpdatingOnAnimationFrame
 {
   if (!_displayLink) {
+    // Setting _currentAnimationTimestamp here is connected with manual triggering of performOperations
+    // in operationsBatchDidComplete. If component is being mount _displayLink.timestamp will be equal to zero and then
```

Comment:
If new node has been created and clock has not been started, _displayLink won't be initialized soon enough and _displayLink.timestamp will be 0.